### PR TITLE
pass browserify-supplied this argument to transformFn in makeStringTrans...

### PR DESCRIPTION
**Pull request by cgc**
...form

I'd need functionality like this to make https://github.com/davidguttman/sassify/pull/25 work. It likely makes sense to add tests?

### CODE REVIEW

File: src/transformTools.coffee [hunk 1]
```diff
@@ -90,7 +90,7 @@ exports.makeStringTransform = (transformName, options={}, transformFn) ->
                     config: configData?.config,
                     opts: configData?.config
                 }
-                transformFn content, transformOptions, (err, transformed) =>
+                transformFn.call @, content, transformOptions, (err, transformed) =>
```
Comment (cgc) [inline src/transformTools.coffee]: might make more sense to pass as in `transformOptions`?

**Comment by cgc** [comment]
ping @benbria

**Comment by jwalton** [comment]
If I don't fix this tomorrow, harass me some more.  :)

**Comment by jwalton** [comment]
Published as 1.4.0.

**Pull request merged**